examples/http-retry/http_retry.bal

```diff
@@ -3,71 +3,66 @@ import ballerina/log;
 import ballerina/runtime;
 
 // Define the end point to the call the `mockHelloService`.
-endpoint http:Client backendClientEP {
-    url: "http://localhost:8080",
-    // Retry configuration options.
-    retryConfig: {
+http:Client backendClientEP = new("http://localhost:8080", config = {
+        // Retry configuration options.
+        retryConfig: {
 
-        // Initial retry interval in milliseconds.
-        interval: 3000,
+            // Initial retry interval in milliseconds.
+            interval: 3000,
 
-        // Number of retry attempts before giving up
-        count: 3,
+            // Number of retry attempts before giving up
+            count: 3,
 
-        // Multiplier of the retry interval to exponentailly
-        // increase; retry interval
-        backOffFactor: 2.0,
+            // Multiplier of the retry interval to exponentailly
+            // increase; retry interval
+            backOffFactor: 2.0,
 
-        // Upper limit of the retry interval in milliseconds
-        // If interval into backOffFactor value exceeded
-        // maxWaitInterval interval values. maxWaitInterval
-        // will be considered as the retry intrval.
-        maxWaitInterval: 20000
+            // Upper limit of the retry interval in milliseconds
+            // If interval into backOffFactor value exceeded
```

Comment:
Same comment regarding single back ticks applies here as well. Please apply to all BBEs.